dashboard/src/components/Catalog/CatalogItems.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { getIcon } from "shared/Operators";
+import { IChart, IClusterServiceVersion } from "shared/types";
+import CatalogItem, { ICatalogItemProps } from "./CatalogItem.v2";
+
+interface ICatalogItemsProps {
+  charts: IChart[];
+  csvs: IClusterServiceVersion[];
+  namespace: string;
+}
+
+export default function CatalogItems({ charts, csvs, namespace }: ICatalogItemsProps) {
+  const chartItems: ICatalogItemProps[] = charts.map(c => {
+    return {
+      type: "chart",
+      id: `chart/${c.attributes.repo.name}/${c.id}`,
+      item: {
+        id: c.id,
+        name: c.attributes.name,
+        icon: c.attributes.icon ? `api/assetsvc/${c.attributes.icon}` : undefined,
+        version: c.relationships.latestChartVersion.data.app_version,
+        description: c.attributes.description,
+        repo: c.attributes.repo,
+        namespace,
+      },
+    };
+  });
```

Comment:
Use `useMemo` here. You don't want to compute this on every render of this component, but only when `charts` changes.